dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java

```diff
@@ -1183,7 +1187,6 @@ public Workflow manualRegister(@ApiParam(hidden = true) @Auth User user, @ApiPar
         final long workflowID = workflowDAO.create(newWorkflow);
         // need to create nested data models
         final Workflow workflowFromDB = workflowDAO.findById(workflowID);
-        workflowFromDB.getUsers().add(user);
```

Comment:
This seems like an odd change because this is data going into the webservice. 
Think we might need a test to ensure that manually registered tools have an owner now.